res/addons/amxmodx/data/lang/bomberman_mod.txt

```diff
@@ -10,8 +10,8 @@ MENU_INTRO_JOIN = Join a room
 MENU_INTRO_SPECT = Join as spectator
 MENU_INTRO_INFO = Information
 
-MENU_CONFIRM_TITLE = \wDo you really want to lave?
-MENU_CONFIRM_YES = Yes, leave room
+MENU_CONFIRM_TITLE = \wDo you really want to leave?
+MENU_CONFIRM_YES = Yes, leave from room
```

Comment:
"leave from room" is not correct. It should be "leave the room".
Please change this so I can approve your PR.